test/integration/serverless/serverless.yml

```diff
@@ -24,27 +24,38 @@ provider:
     DD_SERVICE: integration-tests-service
     DD_TAGS: tagA:valueA tagB:valueB
     DD_EXTRA_TAGS: tagC:valueC tagD:valueD
+    DD_ARCHITECTURE: ${env:ARCHITECTURE}
```

Comment:
What does this env var do?

Reply:
reads environment variables passed into the serverless deploy command

Reply:
in this case DD_ARCHITECTURE will take on the architecture value is passed into our ARCHITECTURE=___ aws-vault exec sandbox-account-admin -- ./run.sh

Reply:
This is a remnant from some debugging oops, let me remove that